redaxo/src/core/pages/packages.list.php

```diff
@@ -21,6 +21,10 @@
     </div>
 ';
 
+$fragment = new rex_fragment();
+$fragment->setVar('id', 'rex-js-available-addon-search');
+$toolbar = $fragment->parse('core/form/searchfield.php');
```

Comment:
Die anderen Dateien in dem Ordner sind auch Felder, haben aber nicht den `field` suffix.
Daher vielleicht besser hier auch schlicht `core/form/search.php`?

Reply:
search fand ich zu allgemein. Es ist ja "nur" ein input. Aber ja gerne können wir machen.